pkg/enqueue/Symfony/Creator/JobCommandCreator.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Enqueue\Symfony\Creator;
+
+use Enqueue\Client\ProducerInterface;
+
+class JobCommandCreator
```

Comment:
Instead of a new layer over producer I would add a RunCommandMessage which accept command name, arguments, options on its constructor, implmenets JsonSerializer interface

```php
<?php

$message = new RunCommandMessage('debug:container', [], ['--env' => 'dev']);

$producer->send(RunCommandProcessor::COMMAND, $message);
```

wdyt?